src/dotnet-install.ps1

```diff
@@ -16,9 +16,18 @@
     - LTS - most current supported release
     - 2-part version in a format A.B - represents a specific release
           examples: 2.0, 1.0
+    - 3-part version in a format A.B.Cxx - represents a specific SDK release
```

Comment:
Why is this possible only since 5.0? And how about A.B.x syntax that AzDo and github actions uses? for example 3.1.x to get the latest 3.1.xxx relelase, is that possible?

Reply:
this is only possible using aka.ms links, but they are not generated for the versions before 5. 
We haven't discussed using 5.0.x format, agreed to use 5.0 instead as before.

Reply:
ok, standardizing probably just leads to a new standard :D